components/AnimatedTaskCard.js

```diff
@@ -76,8 +76,7 @@ const styles = StyleSheet.create({
   taskCompletedIcon: {
     marginTop: 6
   },
-  moveText: {
-    color: "white",
-    marginLeft: 24
```

Comment:
This was left over from refactor shuffling around and is no longer being used in this file